pygohcl.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+// typedef struct {
+// char *json;
+// char *err;
+// } parseResponse;
+import "C"
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclparse"
+)
+
+//export Parse
+func Parse(a *C.char) C.parseResponse {
+	input := C.GoString(a)
+	hclFile, diags := hclparse.NewParser().ParseHCL([]byte(input), "tmp.hcl")
```

Comment:
@lbn leftover: usually we deal with files, but the only interface `pygohcl` has is `.loads(...)`, so the original filename disappears here, and Go layer assigns a placeholder name `tmp.hcl` to the payload, and HCLParseError confusedly carries it

```
HCLParseError: invalid HCL: tmp.hcl:1,1-2: Invalid character; Single quotes are not valid. Use double quotes (") to enclose strings., tmp.hcl:1,1-2: Argument or block definition required; An argument or block definition is required here.
```